README.md

```diff
@@ -270,12 +270,12 @@ See code for all available configurations.
 | [MSI GS60 2QE](msi/gs60)                                               | `<nixos-hardware/msi/gs60>`                             |
 | [MSI GL62/CX62](msi/gl62)                                              | `<nixos-hardware/msi/gl62>`                             |
 | [Microchip Icicle Kit](microchip/icicle-kit)                           | `<nixos-hardware/microchip/icicle-kit>`                 |
-| [Microsoft Surface Go](microsoft/surface/surface-go)                   | `<nixos-hardware/microsoft/surface/surface-go>`         |
```

Comment:
When we remove old models, we put deprecation warnings in place so that existing users now where there model went: https://github.com/NixOS/nixos-hardware/blob/master/microsoft/surface/default.nix

Reply:
There is also flake.nix that needs to be updated.

Reply:
@Mic92 gotchya, I believe it is fixed now. thank you!

Reply:
There are still references missing here: https://github.com/NixOS/nixos-hardware/blob/cde8f7e11f036160b0fd6a9e07dc4c8e4061cf06/flake.nix#L208
Also `/microsoft/surface/surface-go` needs warning.

Reply:
okay, so all changed directories need deprecation messages, not just module references?

Reply:
@Mic92 i only ask because i changed a good number of paths, but ensured that all modules in the flake point to the right place, including the surface go. i made just about the same changes to the surface go as i did the surface laptop amd, and the paths to everything in the pro series was changed. only model removed was sp3.

Reply:
Everything we had before in flake.nix and the README should still work or throw a warning. I personally would not do to many refactoring that just provide minor cosmetic improvements. We usually do these rename if we split up hardware profiles into multiple devices or if we unify things or if models where simply named incorrectly.

Reply:
> Everything we had before in flake.nix and the README should still work or throw a warning.

That is true right now, Surface Pro 3 configuration is the only one that should throw an error.

> I personally would not do to many refactoring that just provide minor cosmetic improvements. We usually do these rename if we split up hardware profiles into multiple devices or if we unify things or if models where simply named incorrectly.

understood. would it be better if i just close this?

Reply:
@Mic92 I'm willing to do whatever, i just wanna understand what you'd like.

Reply:
Sorry. I am busy with other things just now. If it can wait a bit, I will have a look later. Apart from that I invite @mexisme to make judgment calls.

Reply:
If you want to speed up the process, please separate fixes from refactorings. Than I can already merge fixes itself.

Reply:
don't worry about it, i'll close this and do the proper things separately. thanks for working through this with me, i apologize for my poor form as this is my first contribution. :pray: